### ISSUE TITLE
Cannot import Quill

### ISSUE THREAD

**Issue by planetmalone** [KillerCodeMonkey/ngx-quill#105]
I ran `npm update` this morning and can no longer import `Quill`. It was working until I updated. My versions of `ngx-quill` and `quill` did not change. My suspicion is a transitive dependency was updated that is not long compatible. 

My code looks like:

```
import Quill from 'quill';
...

@Component(...)
export class EditorComponent {

    constructor() {
        const icons = Quill.import('./icons');
        ...
    }
}
```

I'm getting the following error:
```
core.js:1350 ERROR Error: Uncaught (in promise): TypeError: Cannot read property 'import' of undefined
TypeError: Cannot read property 'import' of undefined
    at new EditorComponent (editor.component.ts:39)
    at createClass (core.js:12161)
    at createDirectiveInstance (core.js:12010)
    at createViewNodes (core.js:13448)
    at callViewAction (core.js:13880)
    at execComponentViewsAction (core.js:13789)
    at createViewNodes (core.js:13476)
    at callViewAction (core.js:13880)
    at execComponentViewsAction (core.js:13789)
    at createViewNodes (core.js:13476)
    at new EditorComponent (editor.component.ts:39)
    at createClass (core.js:12161)
    at createDirectiveInstance (core.js:12010)
    at createViewNodes (core.js:13448)
    at callViewAction (core.js:13880)
    at execComponentViewsAction (core.js:13789)
    at createViewNodes (core.js:13476)
    at callViewAction (core.js:13880)
    at execComponentViewsAction (core.js:13789)
    at createViewNodes (core.js:13476)
    at resolvePromise (zone.js:783)
    at resolvePromise (zone.js:754)
    at eval (zone.js:831)
    at ZoneDelegate.invokeTask (zone.js:424)
    at Object.onInvokeTask (core.js:4617)
    at ZoneDelegate.invokeTask (zone.js:423)
    at Zone.runTask (zone.js:191)
    at drainMicroTaskQueue (zone.js:595)
    at ZoneTask.invokeTask [as invoke] (zone.js:502)
    at invokeTask (zone.js:1364)
```

I tried importing some of the other exports from `quill`, but the error was given.

Thanks in advance for any help.

**Comment by KillerCodeMonkey** [comment]
Aaaand what can i do for it? Checkout quilljs typings if there is anything
wrong.

Which versions are you using?
I have to dirty use quilljs in my repo to avoid those errors during build.

Am 22.01.2018 10:21 nachm. schrieb "Sean Malone" <notifications@github.com>:

**Comment by planetmalone** [comment]
ngx-quill: 2.0.4
quill: 1.3.5

I'm not sure what you mean by dirty use. We'd prefer not to have to pull in quill directly, so that we don't have to maintain it separately from ngx-quill. 

I'll check out the typings repo to see if anyone has posted something there. The Quill library was quiet.

**Comment by KillerCodeMonkey** [comment]
i used need to ugly import quill

```
import * as QuillNamespace from 'quill';
let Quill: any = QuillNamespace;
```

because the typings are not working.

**Comment by KillerCodeMonkey** [comment]
Hey there, i tested with latest quill and latest @types/quill.
I got the same errors if i try to register a custom module.

So i set a fixed quill and typings version

**Comment by planetmalone** [comment]
Thanks for looking into it.

**Comment by johnking** [comment]
@KillerCodeMonkey 
I have the same issue as you mentioned. I found the `ngx-quill` library is using the fixed typings: 1.3.3, but my app is using quill@1.3.5 and typings: 1.3.5. 

how to fix it?
